pkg/kubectl/cmd/BUILD

```diff
@@ -114,6 +114,7 @@ go_library(
         "//vendor:k8s.io/apimachinery/pkg/util/intstr",
         "//vendor:k8s.io/apimachinery/pkg/util/json",
         "//vendor:k8s.io/apimachinery/pkg/util/jsonmergepatch",
+        "//vendor:k8s.io/apimachinery/pkg/util/mergepatchutil",
```

Comment:
Change to `pkg/util/mergepatch`, since `patch` has been used as a variable everywhere.